### ISSUE TITLE
Uncaught RangeError: Trying to access beyond buffer length

### ISSUE THREAD

**Issue by KyleMaas** [arj03/ssb-browser-demo#317]
Error during sync, after previously refreshing the page mid-sync:

```
Uncaught RangeError: Trying to access beyond buffer length
    at checkOffset (bundle-ui.js:53264)
    at Uint8Array.readUInt16LE (bundle-ui.js:53306)
    at Object.getDataNextOffset (bundle-ui.js:25410)
    at Stream._handleBlock (bundle-ui.js:25674)
    at Stream._resumeCallback (bundle-ui.js:25745)
    at Object.getBlock (bundle-ui.js:25364)
    at Stream._resume (bundle-ui.js:25736)
    at Stream._next (bundle-ui.js:80406)
    at Stream.resume (bundle-ui.js:25756)
    at bundle-ui.js:25517
```

**Comment by KyleMaas** [comment]
Tried refreshing the page again.  Same error.

**Comment by KyleMaas** [comment]
And I just got this `alert()` box:

`An exception was encountered trying to read the messages database.  Please report this so we can try to fix it: RangeError: Trying to access beyond buffer length`

**Comment by KyleMaas** [comment]
Yep, that seems to have just completely broken that instance.  Can't load the Public tab anymore.  It just sits there with the loading indicator up and never goes anywhere.

@arj03 Is there anything in that instance you'd need to figure out what's going on, or is it safe to clear that one out and rebuild it?

**Comment by arj03** [comment]
oh that is pretty bad. I'll try and see if I can reproduce, sadly we don't have the option of extracting the db.

**Comment by KyleMaas** [comment]
I'll note that I still get EBT debugging messages logged to the console (I have that enabled).  So that's at least still trying to operate.  But I can't access any messages.

**Comment by KyleMaas** [comment]
Are we in a spot where I can wipe this instance or should I keep it around yet?

**Comment by arj03** [comment]
Wiping is fine
